requirements.txt

```diff
@@ -1,3 +1,6 @@
 numpy>=1.9.0
 scikit-learn>=0.22.2
 scipy>=0.19.1
+uvicorn>=0.14.0
+fastapi>=0.68.0
+requests>=2.0.0
```

Comment:
I was thinking maybe we can add this file to another requirements file.
What do you think @sepandhaghighi ?

Reply:
hmmm, I agree, I will do that.

Reply:
applied.